scripts/lib/mkcloud-dirmaint.sh

```diff
@@ -19,57 +21,77 @@ function dirmaint_do_sanity_checks()
     fi
 }
 
+function dirmaint_user_deleted()
+{
+    vmcp q $1 2> /dev/null | grep HCPCQV003E >& /dev/null
```

Comment:
&> or >& are identical in this respect. The difference seems to be, that with &>> you would be able to append to a file.